.binder/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+mkdir -p ~/.data/db
+# Start mongodb server
+nohup ~/.tools/mongodb/bin/mongod --dbpath ~/.data/db > mongo.txt 2>&1 &
+sleep 5
+
+# Restore database
+~/.tools/mongotools/bin/mongorestore ~/dump
```

Comment:
```suggestion
~/.tools/mongotools/bin/mongorestore ~/dump
rm -rf ~/dump
```
I _think_ that I can do this? If so, I would like to remove as it just looks 'cleaner' to a user when they start a session.

Reply:
If you rm dump 
we can't use it again :)